pkg/kubelet/cm/deviceplugin/pod_devices.go

```diff
@@ -252,6 +255,20 @@ func (pdev podDevices) deviceRunContainerOptions(podUID, contName string) *Devic
 				SELinuxRelabel: false,
 			})
 		}
+
+		// Updates for Annotations
+		for k, v := range resp.Annotations {
+			if e, ok := annotationsMap[k]; ok {
+				glog.V(4).Infof("skip existing annotation %s %s", k, v)
```

Comment:
nit: AFIK we should start with upper case for log msg, and lower case for error msg. Not sure why it's in the other way in this file.  -_-!

Reply:
I'll update that for all log messages